src/include/sof/trace/trace.h

```diff
@@ -387,5 +412,4 @@ do {									\
 						##__VA_ARGS__)
 #define trace_error_comp(...) _trace_comp_build(trace_error_with_ids,	\
 						##__VA_ARGS__)
-
 #endif /* __SOF_TRACE_TRACE_H__ */
```

Comment:
If you want to remove this line, you need to do it in a separate cleanup patch.

Reply:
Updated other components and removed that old block completely.